source/a_box/tmu/tmu.c

```diff
@@ -125,14 +126,13 @@ void readTemps(tmu_handle_t *tmu) {
         int16_t min_temp =  MIN(MIN(tmu->tmu1_min, tmu->tmu2_min), MIN(tmu->tmu3_min, tmu->tmu4_min));
         SEND_MAX_CELL_TEMP(max_temp);
         SEND_NUM_THERM_BAD(num_bad1, num_bad2, num_bad3, num_bad4);
-        num_bad1 = 0;
-        num_bad2 = 0;
-        num_bad3 = 0;
-        num_bad4 = 0;
         setFault(ID_PACK_TEMP_FAULT, max_temp);
         setFault(ID_PACK_TEMP_EXCEEDED_FAULT, max_temp);
         setFault(ID_MIN_PACK_TEMP_FAULT, min_temp);
 
+        // setting temp error fault high for bms fault if there is temp fault
+        tempError = max_temp > MAX_TEMP_TMU ? 1 :  min_temp < MIN_TEMP_TMU ? 1 : 0;
```

Comment:
after calling setFault use checkFault to get if the fault was latched so that the values and settings in fault_config are what control this (do this for the error ones such as temp exceeded).
Will this fault if a thermistor becomes disconnected?

Reply:
The fault library will also provide the latching time so that it has to exceed the value for an amount of time before latching to prevent it from faulting on noise.

Reply:
added the check faults. It depends where thermistor disconnects I guess, since one will lead to a pull up and the other a pull down, but yeah in other case there would be a fault. We'd know though because it will send out either INT MAX or INT MIN when it disconnects, which is how we would differentiate ig